data/accounts.json

```
@@ -2,6 +2,7 @@
   {
     "id": "4",
     "name": "The Beatles",
+    "mrr": "398",
```

```suggestion
    "mrr": 398,
```
Can send it as an actual number ^^